test/react-native/features/steps/react-native-steps.rb

```diff
@@ -1,68 +1,99 @@
-When("I run {string}") do |event_type|
+def wait_for_true
+  # max_attempts = 300
+  max_attempts = 100
+  attempts = 0
+  assertion_passed = false
+  until (attempts >= max_attempts) || assertion_passed
+    attempts += 1
+    assertion_passed = yield
+    sleep 0.1
+  end
+  # raise 'Assertion not passed in 30s' unless assertion_passed
```

Comment:
I left this in by mistake, but will fix on my next branch (in progress now) to avoid a whole new build, if possible.